### ISSUE TITLE
Make timestamp a required field in ODIS requests

### ISSUE THREAD

**Issue by alecps** [celo-org/celo-monorepo#6825]
We have implemented protections against replay attacks in ODIS but they are currently disabled because we are not requiring that requests have a timestamp. This requirement was originally postponed for backwards compatibility reasons but it is now safe to assume all clients are submitting requests with a timestamp so we should impose this requirement.

**Comment by codyborn** [comment]
Can be done with #6824

**Comment by alecps** [comment]
Flagging this as potential cross bubble work as all ODIS clients will need to provide timestamps for this change to be backwards compatible.

**Comment by nambrot** [comment]
Should we split up this ticket to be 1) investigating the clients right now and then 2) making it required? I assume 2) is blocked on 1) cc @aslawson

**Comment by codyborn** [comment]
@alecps is the concern here for compatibility with non-Valora clients of ODIS?

**Comment by alecps** [comment]
Looking at the logs I think it's safe to assume all clients are submitting requests with a timestamp at this point. Unless there are objections I think we can update the ticket accordingly

**Comment by alecps** [comment]
We have opted to remove timestamps due to a synchronization error. We will now handle replays as outlined in this ticket https://app.zenhub.com/workspaces/cap-backlog-board-600598462807be0011921c65/issues/celo-org/celo-monorepo/7798
